classes/edge.bbclass

```diff
@@ -46,6 +46,9 @@ EDGE_ETC = "/wigwag/system/etc"
 EDGE_DATA = "/userdata"
 EDGE_STATE = "/wigwag/system/var/lib"
 EDGE_OPT = "/wigwag/system/opt"
+EDGE_CONFIG = "/wigwag/etc/run"
```

Comment:
At least EDGE_CONFIG needs to be added to the edge_replace_vars() function below. 

Maybe EDGE_TEMPLATE and EDGE_LOG as well